modules/aws/awc/main.tf

```diff
@@ -70,6 +71,13 @@ resource "aws_s3_object" "tls-cert" {
   source = var.tls_cert
 }
 
+# EditShare-specific
+resource "aws_s3_object" "log-manager-script" {
+  bucket = var.bucket_name
+  key    = local.log_manager_script
+  source = "${path.module}/../../../../editshare_deployments/modules/connector/scripts/${local.log_manager_script}"
```

Comment:
_Just a cross reference mention here to comment about mixed `local`/`var` usage on line [13](https://github.com/EditShareCo/cloud_deployment_scripts_hp_inc/pull/5/files#diff-51d281d110c60958957a4a5e352dd6e2aa0aa02de13680b3df9564a0dfbe6852R13)_

Reply:
general note that the use of var.log_manager_script has been removed, it was inconsistent with the way the rest of the script names were managed by the original teradici code, and caused more problems/changes than it was worth.  there should now only be references to local.log_manager_script, or to the underlying file name which is connector_log_manager.py

Reply:
👍 Seems sensible =)